### DISCUSSION TITLE
Everithing is messed up!

### DISCUSSION THREAD

**Discussion by Theoyeah** [DEMOLITIONDON96/Demolition-Engine#106] [General (Friday Night Chattin')]
![image](https://user-images.githubusercontent.com/97792861/170456823-59c6ad9c-3cbf-48dd-b8f0-138e4c74feb1.png)
Wtf did you do @DEMOLITIONDON96

**Comment by ThatOneSillyGuyDon** [comment]
fuck, i forgot about the project file XD

**Reply by Theoyeah** [reply]
+ Wtf happened to the sustains  notes
